packages/blockchain-extension/test/commands/UserInputUtil.test.ts

```diff
@@ -2715,6 +2715,25 @@ describe('UserInputUtil', () => {
         });
     });
 
+    describe('failedNetworkStart', () => {
+
+         it('should display a message passed and the More Details button', async () => {
+            const showErrorMessageStub: sinon.SinonStub = mySandBox.stub(vscode.window, 'showErrorMessage').resolves();
+            const outputAdapter: VSCodeBlockchainOutputAdapter = VSCodeBlockchainOutputAdapter.instance();
+            await UserInputUtil.failedNetworkStart('some error', 'some error message', outputAdapter);
+            showErrorMessageStub.should.have.been.calledOnceWithExactly('some error', 'More Details');
```

Comment:
As the `showErrorMessageStub` stub is just resolving (which means that the user has cancelled / closed the popup), this means that `outputAdapter.show();` hasn't been called.

So you could do something like:
```
showStub = mySandBox.stub(VSCodeBlockchainOutputAdapter.instance(), 'show');

await UserInputUtil.failedNetworkStart('some error', 'some error message');

showStub.should.not.have.been.called;
```

We should be as strict as possible in our tests!